### PR TITLE
[Doc] Update docstrings for functions in ops

### PR DESCRIPTION
**Pull request by neozhaoliang**

### CODE REVIEW

File: python/taichi/lang/ops.py [hunk 1]
```diff
@@ -1125,45 +1125,220 @@ def py_select(cond, x1, x2):
...
+        The old value of `x`.
+
+    Example::
+
+        >>> @ti.kernel
+        >>> def test():
+        >>>     x = ti.Vector([-1, 0, 1])
+        >>>     y = ti.Vector([1, 2, 3])
+        >>>     z = ti.atomic_and(x, y)
+        >>>     print(x)  # [1, 0, 0]  the new value of x
```
Comment (rexwangcc) [inline python/taichi/lang/ops.py:1273]:
```suggestion
        >>>     print(x)  # [1, 0, 1]  the new value of x
```
`1 && 3` should produce `[1, 0, 1]`

File: python/taichi/lang/ops.py [hunk 2]
```diff
@@ -1125,45 +1125,220 @@ def py_select(cond, x1, x2):
...
+
+        >>> @ti.kernel
+        >>> def test():
+        >>>     x = ti.Vector([-1, 0, 1])
+        >>>     y = ti.Vector([1, 2, 3])
+        >>>     z = ti.atomic_xor(x, y)
+        >>>     print(x)  # [-2, 2, 2]  the new value of x
+        >>>     print(z)  # [-1, 0, 1], the old value of x
+        >>>
+        >>>     ti.atomic_and(1, x)  # will raises TaichiSyntaxError
```
Comment (rexwangcc) [inline python/taichi/lang/ops.py:1338]:
```suggestion
        >>>     ti.atomic_xor(1, x)  # will raises TaichiSyntaxError
```
typo?

File: python/taichi/lang/ops.py [hunk 3]
```diff
@@ -1174,6 +1349,28 @@ def assign(a, b):
...
+        Maximum of the inputs.
+
+    Example::
+
+        >>> @ti.kernel
+        >>> def foo():
+        >>>     x = ti.Vector([0, 1, 2])
+        >>>     y = ti.Vector([3, 4, 5])
+        >>>     z = ti.max(x, y, 4)
+        >>>     print(z)  # [4, 4, 5]
```
Comment (rexwangcc) [inline python/taichi/lang/ops.py:1372]: Not sure if really works currently :thinking:

**Review by rexwangcc — CHANGES_REQUESTED** [review]
Thanks for the work!! :rocket: I left my 2 cents.

File: python/taichi/lang/ops.py (hunk 2 above)
Comment (neozhaoliang) [reply python/taichi/lang/ops.py:1338]: Yes, it's a copy-paster typo :laughing: :

File: python/taichi/lang/ops.py (hunk 3 above)
Comment (neozhaoliang) [reply python/taichi/lang/ops.py:1372]: You may this behavior has changed recently?

Comment (rexwangcc) [reply python/taichi/lang/ops.py:1372]: Yeah, I haven't use this func in a while so just built Taichi from source, the above code does not work well on the current `master` branch. Maybe I missed something?

Comment (rexwangcc) [reply python/taichi/lang/ops.py:1372]: Per offline discussion, I was using 0.8.8 at this place, NVM.

**Review by rexwangcc — APPROVED** [review]
LGTM!

**Pull request merged**